test/spec/unit/configparser.js

```diff
@@ -9,7 +9,8 @@ describe('ConfigParser', () => {
         configParser.addConfigFile(FIXTURES_PATH + '/exclude.conf.js')
         let specs = configParser.getSpecs()
         specs.should.not.include(__filename)
-        specs.should.include(__dirname + '/call.js')
+        specs.should.include(path.resolve(__dirname + '/call.js'))
+        specs.should.include(path.resolve(__dirname + '/typescript.ts'))
```

Comment:
Would you be opposed to an exclude3.conf.js that handles this? I can create a new test that uses exclude3 and revert this one back to the original.